Sources/Extensions/Watch/ComplicationController.swift

```diff
@@ -41,7 +41,25 @@ class ComplicationController: NSObject, CLKComplicationDataSource {
         for complication: CLKComplication,
         withHandler handler: @escaping (CLKComplicationPrivacyBehavior) -> Void
     ) {
-        handler(.showOnLockScreen)
+
+        let model: WatchComplication?
+
+        if #available(watchOS 7, *), complication.identifier != CLKDefaultComplicationIdentifier {
```

Comment:
given this is from another method in this class, i think it's worth writing a helper function like `func complication(for complication: WatchComplication) -> WatchComplication?`